packages/beacon-utils/package.json

```diff
@@ -38,6 +38,7 @@
     "@stablelib/nacl": "^1.0.4",
     "@stablelib/random": "^1.0.2",
     "@stablelib/utf8": "^1.0.1",
+    "@taquito/utils": "^17.0.0",
```

Comment:
You're right, better to avoid adding one more for 1 function

Reply:
I replaced it with an inlined method. However, I'm wondering if we do want to have a strict address check. In case that a new address type comes out (eg. tz5), it would mean that old dApps would error if a new address is shared. Maybe that's a bit too much?